README.md

```diff
@@ -1,3 +1,6 @@
+# Other Languages
```

Comment:
H1 is the wrong thing to use here. I'd suggest doing something like:

```md
# Microsoft Python Language Server

[Português - Brasil](README.pt-br.md)

Microsoft Python Language Server implements ...
```

Reply:
Alright, I'll do it now